lib/auth/apiserver.go

```diff
@@ -751,19 +751,41 @@ func (s *APIServer) u2fSignRequest(auth ClientI, w http.ResponseWriter, r *http.
 	return u2fSignReq, nil
 }
 
-type createWebSessionReq struct {
-	PrevSessionID   string `json:"prev_session_id"`
+// CreateWebSessionReq defines fields for requesting to create new web session.
+type CreateWebSessionReq struct {
```

Comment:
This is used to extend a web session, `ExtendWebSessionRequest` could be a more apt name.